worldedit-core/src/main/java/com/sk89q/worldedit/LocalSession.java

```diff
@@ -885,25 +903,38 @@ public Calendar detectDate(String input) {
     /**
      * Construct a new edit session.
      *
-     * @param player the player
+     * @param actor the actor
      * @return an edit session
      */
-    public EditSession createEditSession(Player player) {
-        checkNotNull(player);
+    public EditSession createEditSession(Actor actor) {
+        checkNotNull(actor);
 
-        BlockBag blockBag = getBlockBag(player);
+        World world = null;
+        if (hasWorldOverride()) {
+            world = getWorldOverride();
+        } else if (actor.isPlayer() && actor instanceof Player) {
```

Comment:
This seems redundant? Is there a case where we have "Player" but not "isPlayer"?

Reply:
Honestly I should just make a LocatableActor interface that Player implements - as CommandBlocks are locatable as well.